src/sentry/snuba/metrics.py

```diff
@@ -470,28 +471,28 @@ class SnubaField:
 )
 
 _METRICS = {
-    "session": {
+    SessionMetricKey.SESSION.value: {
```

Comment:
I would recommend here not adding the `.value` and just using the enum

Reply:
I like the idea, but this dict also contains metric names that are not session metrics, so I think we should stick with strings here.